src/main/drivers/pwm_output.c

```diff
@@ -373,6 +385,125 @@ uint32_t getDshotHz(motorPwmProtocolTypes_e pwmProtocolType)
         return MOTOR_DSHOT150_HZ;
     }
 }
+
+void pwmWriteDshotCommand(uint8_t index, uint8_t motorCount, uint8_t command, bool blocking)
+{
+    timeUs_t timeNowUs = micros();
+    timeUs_t timeDelayUs;
+    if (isMotorProtocolDshot() && (command <= DSHOT_MAX_COMMAND)) {
+        uint8_t repeats;
```

Comment:
Seems like this block could be unindented by inverting the condition and returning.

Reply:
The idea is avoid the indent and put the "error" condition check right away? I put that change in as well